data/simpy.gram

```diff
@@ -427,7 +427,21 @@ star_target: '*' bitwise_or | atom t_tail+ | a=star_atom t_tail* { a }
 star_atom: a=NAME { set_expr_context(p, a, Store) } | '(' [star_targets] ')' | '[' [star_targets] ']'
 
 targets[asdl_seq*]: a=','.target+ [','] { a }
-target[expr_ty]: atom t_tail+ | a=t_atom t_tail* { a }
+target[expr_ty]:
+    | a=non_terminal_target '.' b=NAME !tmore { _Py_Attribute(a, b->v.Name.id, Load, EXTRA) }
+    | a=non_terminal_target b=slicing !tmore { _Py_Subscript(a, b, Load, EXTRA) }
+    | t_atom
+non_terminal_target[expr_ty]:
```

Comment:
While I agree that `target1` was a bad name showing lack of imagination, `non_terminal_target` feels too long. How about `t_primary`? Because it's exactly the same as `primary` except for the lookaheads, and we already have `t_atom` that's the target equivalent of `atom`.